tests/test_workers/test_tasks/test_fbc_utils.py

```diff
@@ -86,3 +88,27 @@ def test_is_image_fbc(mock_si, skopeo_output, is_fbc):
 
     mock_si.return_value = skopeo_output
     assert is_image_fbc(image) is is_fbc
+
+
+@mock.patch('shutil.copytree')
+@mock.patch('os.path.isdir')
+def test_merge_catalogs_dirs(mock_isdir, mock_cpt, tmpdir):
```

Comment:
We could test here that copytree works as expected and the permissions on the dirs are not changed when overwritten.